tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -517,6 +517,66 @@ function test_returns_post_object_by_id() {
 		$this->assertEquals( $synced_post, $retrieved_post );
 	}
 
+	function test_remove_contact_form_shortcode_from_fitered_content() {
+		$this->post->post_content = '<p>This post has a contact form:[contact-form][contact-field label=\'Name\' type=\'name\' required=\'1\'/][contact-field label=\'Email\' type=\'email\' required=\'1\'/][contact-field label=\'Website\' type=\'url\'/][contact-field label=\'Comment\' type=\'textarea\' required=\'1\'/][/contact-form]</p>';
+
+		Grunion_Contact_Form_Plugin::init();
+
+		wp_update_post( $this->post );
+		$this->sender->do_sync();
+
+		$synced_post = $this->server_replica_storage->get_post( $this->post->ID );
+
+		$this->assertEquals( "<p>This post has a contact form:</p>\n", $synced_post->post_content_filtered );
+	}
+
+	function test_remove_likes_from_fitered_content() {
```

Comment:
This test could use a positive test before we do a negative one - i.e. render the filtered content for the plugin, assert that it contains the likes HTML, then have the rest of the test which asserts that the likes HTML doesn't get synced.

Also `test_remove_likes_from_fitered_content` should be `test_remove_likes_from_filtered_content`